hadoop-ozone/dist/src/main/smoketest/basic/ozone-shell-lib.robot

```diff
@@ -188,6 +186,8 @@ Test key handling
                     Should Contain      ${result}       NOTICE.txt.1 exists
     ${result} =     Execute             ozone sh key get --force ${protocol}${server}/${volume}/bb1/key1 /tmp/NOTICE.txt.1
                     Should Not Contain  ${result}       NOTICE.txt.1 exists
+    ${result} =     Execute and checkrc    ozone sh key put ${protocol}${server}/${volume}/bb1/key1 sample.txt          255
+                    Should Match Regexp    ${result}       Error: File not found: .*
```

Comment:
I guess we can make this assertion a bit more specific by changing to `Should Contain` and including `sample.txt` in the expected message.